core/src/main/java/org/akaza/openclinica/web/filter/ApiSecurityFilter.java

```diff
@@ -239,8 +249,8 @@ public OCUserDTO getUserDetails (HttpServletRequest request) {
     }
 
     //TODO Put this somewhere else?
-    private Map<String, String>  createRandomizeUserAccount() throws Exception {
-        Map<String, String> map = new HashMap<>();
+    private HashMap<String, String>  createRandomizeUserAccount() {
```

Comment:
Same reason as before - I had to use HashMap because userService.createUser method was expecting a HashMap. We were casting Map back to HashMap so I changed it to be HashMap